### PR TITLE
Add korean translation.

### PR DESCRIPTION
**Pull request by 10sa**
:P

(Redirect from https://github.com/HaodongMo/ArcCW/pull/37#issuecomment-723743801)

### CODE REVIEW

**Comment by TheOnly8Z** [comment]
It seems like you translated the client convars, but used the format of custom language instead. This will unfortunately not work (for menus, we rely on gmod's stock language system).

I'll write up an example and link it here in a bit.

**Comment by TheOnly8Z** [comment]
Please set your file as `arccw/client/cl_languages/[LANG_NAME].lua`. See the repo for my example.

**Comment by 10sa** [comment]
@TheOnly8Z Done.

File: lua/arccw/client/cl_languages/ko.lua [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+-- TTT Menus
+L["arccw.cvar.attinv_loseondie.help"]     = "활성화된 경우 사망 또는 라운드 종료 시 모든 부착물을 제거합니다."
+L["arccw.cvar.ammo_detonationmode.help"]  = "탄약 상자 파괴 시 폭발할지에 대한 여부입니다."
+L["arccw.cvar.equipmenttime.help"]        = "크레모아 같은 전개 가능한 장비에 적용됩니다."
+L["arccw.cvar.ttt_bodyattinfo"]           = "시체 공격 부착물 정보"
+L["arccw.cvar.ttt_bodyattinfo.help"]      = "활성화 시 시체 수색이 어떠한 부착물을 장비한 무기에 의해 사망하였는지 표시합니다."
+L["arccw.cvar.ttt_bodyattinfo.desc"]      = "0 - 끄기; 1 - 탐정만 볼 수 있음; 2 - 모두가 볼 수 있음"
+L["arccw.cvar.attinv_free.help"]          = "활성화 시 모든 부착물을 사용할 수 있습니다. 커스터마이징 모드에 따라 제한될 수 있습니다."
+L["arccw.cvar.attinv_lockmode.help"]      = "활성화 시 습득한 부착물이 모든 무기에서 해금됩니다."
+L["arccw.cvar.ttt_weakensounds"]          = "낮은 발포음"
```
Comment (TheOnly8Z) [inline lua/arccw/client/cl_languages/ko.lua:210]: No longer used - can be removed.

File: lua/arccw/client/cl_languages/ko.lua [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+L["arccw.cvar.mult_range"]                = "사거리"
+L["arccw.cvar.mult_recoil"]               = "반동"
+L["arccw.cvar.mult_penetration"]          = "관통력"
+L["arccw.cvar.mult_hipfire"]              = "비조준 확산"
+L["arccw.cvar.mult_movedisp"]             = "이동 확산"
+L["arccw.cvar.mult_reloadtime"]           = "재장전 시간"
+L["arccw.cvar.mult_sighttime"]            = "정조준 시간"
+L["arccw.cvar.mult_defaultclip"]          = "기본 탄약"
+L["arccw.cvar.mult_attchance"]            = "랜덤 부착물 확률"
+
```
Comment (TheOnly8Z) [inline lua/arccw/client/cl_languages/ko.lua:155]: Missing `L["arccw.cvar.mult_heat"]                 = "Heat Gain"`

File: lua/arccw/client/cl_languages/ko.lua [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+-- ArcCW_Options_Dev
+L["arccw.dev_info1"]                      = "부착물과 관련된 부분으로 개발자만이 사용할 수 있습니다."
+L["arccw.dev_info2"]                      = "설정은 저장되지 않습니다."
+L["arccw.cvar.dev_reloadonadmincleanup"]  = "관리자 클린업 시 리로드"
+L["arccw.cvar.dev_reloadonadmincleanup.desc"]    = "기본 값 : 비활성화\n관리자 클린업이 모든 부착물 데이터를 리로드합니다."
+L["arccw.cvar.dev_registerentities"]      = "부착물 엔티티 등록"
+L["arccw.cvar.dev_registerentities.desc"] = "기본 값 : 활성화\nArcCW 부착물의 엔티티를 생성합니다. 부착물 데이터 리로드 시간이 증가합니다."
+L["arccw.cvar.dev_reloadatts"]            = "부착물 데이터 리로드"
+L["arccw.cvar.dev_reloadatts.desc"]       = "부착물 데이터를 리로드합니다. 부착물 수정 후 사용하세요."
+
```
Comment (TheOnly8Z) [inline lua/arccw/client/cl_languages/ko.lua:200]: Missing `L["arccw.cvar.dev_reloadlangs"]            = "Reload Languages"` and 
`L["arccw.cvar.dev_reloadlangs.desc"]       = "Reloads all language files."`

**Comment by 10sa** [comment]
@TheOnly8Z Done.

File: lua/arccw/client/cl_languages/ko.lua (hunk 3 above)
Comment (10sa) [reply lua/arccw/client/cl_languages/ko.lua:203]: Added new translate texts.

File: lua/arccw/client/cl_languages/ko.lua (hunk 2 above)
Comment (10sa) [reply lua/arccw/client/cl_languages/ko.lua:156]: Add new translate texts.

File: lua/arccw/client/cl_languages/ko.lua (hunk 1 above)
Comment (10sa) [reply lua/arccw/client/cl_languages/ko.lua]: Deleted.

**Pull request merged**